src/Standard.Licensing/Security/Cryptography/KeyFactory.cs

```diff
@@ -43,14 +48,30 @@ internal static class KeyFactory
         /// <returns>The encrypted private key.</returns>
         public static string ToEncryptedPrivateKeyString(AsymmetricKeyParameter key, string passPhrase)
         {
-            var salt = new byte[16];
-            var secureRandom = SecureRandom.GetInstance("SHA256PRNG");
-            secureRandom.SetSeed(secureRandom.GenerateSeed(16)); //See Bug #135
-            secureRandom.NextBytes(salt);
+            // Rent a buffer for the salt (16 bytes for the salt)
+            var salt = ArrayPool<byte>.Shared.Rent(16);
```

Comment:
Potential issue here as the salt array is no longer guaranteed to be 16 bytes long.